### PR TITLE
Move age to a command line argument

### PR DESCRIPTION
**Pull request by rmgr**
I'm running an Impossible Landscapes game and I figured 50+ year old agents in 1995 are going to be pushing 70 in 2015 so I decided to add a couple of command line arguments (-a/--min-age and -A/--max-age).

### CODE REVIEW

File: generator.py [hunk 1]
```diff
@@ -750,6 +751,8 @@ def get_options():
         default="data/professions.json",
         help="Data file for professions - defaults to %(default)s",
     )
+    parser.add_argument("-A", "--max-age", action="store", type=int, help="Maximum age of characters", default=55)
+    parser.add_argument("-a", "--min-age", action="store", type=int, help="Maximum age of characters", default=25)
```
Comment (brunns) [inline generator.py]: Help text needs a tweak.

**Comment by brunns** [comment]
Help text aside, LGTM.

File: generator.py (hunk 1 above)
Comment (rmgr) [reply generator.py]: Damn it, I was working quickly on a bus but I thought I'd tidied up all my mistakes! :)

**Comment by jimstorch** [comment]
Thanks for the contribution!

**Comment by brunns** [comment]
This would work well with #16 if we can work out how we'd like to add skill points to older characters...

**Pull request merged**